code/datums/martial/buster_style.dm

```diff
@@ -156,59 +149,51 @@
 		I.forceMove(Z)
 		I.density = FALSE 
 		walk_towards(I, user, 0, 0)
-		// Reset the item to its original state
 		if(get_dist(I, user) > 1)
 			I.density = old_density
-		thrown |= I // Mark the item for throwing
+		thrown = I 
 		if(ismecha(I))
 			I.anchored = TRUE
 		return COMSIG_MOB_CANCEL_CLICKON
 	if(isliving(target))
 		var/mob/living/L = target
-		var/obj/structure/bed/grip/F = new(Z, user) // Buckles them to an invisible bed
+		var/obj/structure/bed/grip/F = new(Z, user) 
```

Comment:
Why remove the comment

Reply:
was removing a bunch of comments i didnt like and forgot to replace this one oops